plugins/phishtank/komand_phishtank/actions/check/action.py

```diff
@@ -1,56 +1,42 @@
-import komand
-from .schema import CheckInput, CheckOutput
+import insightconnect_plugin_runtime
 
-# Custom imports below
-import requests
+from .schema import CheckInput, CheckOutput, Component, Input, Output
 
 
-class Check(komand.Action):
+class Check(insightconnect_plugin_runtime.Action):
     def __init__(self):
         super(self.__class__, self).__init__(
             name="check",
-            description="Submit URL to Phishtank",
+            description=Component.DESCRIPTION,
             input=CheckInput(),
             output=CheckOutput(),
         )
 
     def run(self, params={}):
-        url = params.get("url")
-        if not url:
-            raise ValueError("url is required")
 
-        url = url.strip()
+        url = params.get(Input.URL, "").strip()
         if not url.startswith("http://") and not url.startswith("https://"):
-            url = "http://" + url
+            url = f"https://{url}"
 
-        try:
-            result = self.connection.check(url)
-            self.logger.debug("result: %s", result)
-        except Exception as e:
-            self.logger.exception(e)
-            return {
-                "url": url,
-                "in_database": False,
-                "verified": False,
-            }
-
-        if "verified_at" in result:
-            if result["verified_at"] is None:
-                result["verified_at"] = str(result["verified_at"])
-
-        return result
+        response = self.connection.phishtank_api.check(url)
+        response = response.get("results")
 
-    def test(self):
-        url = "http://cielobbfidelidade.xyz/home/"
+        if "verified_at" in response:
+            if response["verified_at"] is None:
+                response["verified_at"] = str(response["verified_at"])
 
-        try:
-            result = self.connection.check(url)
-        except Exception as e:
-            self.logger.exception(e)
+        if not response.get("in_database"):
```

Comment:
I think these couple of conditions could be simplified to not access the key/values as many times. something along the lines:
```
in_database = response.get("in_database")
if not in_database:
  return {...}
return {
   Output.IN_DATABASE: in_database,
   .......
   Output.VERIFIED_AT: response.get("verified_at", ""). # this will mean if verified_at is returned as None we just return an empty string to the user instead of "None"

}

```